[IMP]-14.0 - sale_commission_formula: Improve usage readme

hieulucky111 opened the pull request:
Improve module usage README

kluna1998 committed:
[IMP] sale_commission_delegated_partner: Improve readme

In response to #333 comment

hieulucky111 committed:
[IMP]-sale_commission_formula: Improve readme

pedrobaeza commented on sale_commission_formula/readme/USAGE.rst:
```diff
@@ -1,3 +1,38 @@
 To use this module, you need to:
 
-* Go to Sales, Commission Types and create a commission with type formula
+Prerequisite: https://github.com/OCA/commission/blob/14.0/sale_commission/readme/CONFIGURE.rst
```
This paragraph should be in INSTALL.rst and you should avoid absolute references, as on next versions, the version change may be forgotten and it will be misleading.

pedrobaeza commented on sale_commission_formula/readme/USAGE.rst:
```diff
@@ -1,3 +1,38 @@
-* Go to Sales, Commission Types and create a commission with type formula
+Prerequisite: https://github.com/OCA/commission/blob/14.0/sale_commission/readme/CONFIGURE.rst
+
+* Go to Sales > Commission Management >  Commission Types and create a commission with new Type "Formula".
```
And this should go on CONFIGURE.rst.

ilyasProgrammer committed:
[IMP] sale_commission_agent_restrict: access updates

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (158 of 158 strings)

Translation: commission-14.0/commission-14.0-sale_commission
Translate-URL: https://translation.odoo-community.org/projects/commission-14-0/commission-14-0-sale_commission/it/

francesco-ooops committed:
Added translation using Weblate (Italian)

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 11.1% (1 of 9 strings)

Translation: commission-14.0/commission-14.0-sale_commission_salesman
Translate-URL: https://translation.odoo-community.org/projects/commission-14-0/commission-14-0-sale_commission_salesman/it/

CoraVanga reviewed (approved):
LGTM

Leonidas-VII reviewed (approved):
I think this readme is good for me, I can read Readme and use the module directly without reviewing code.

pedrobaeza commented:
My comments hasn't been correctly achieved. Please read them again and put the texts in the corresponding files. You have reversed what should be in INSTALL and in CONFIGURE.

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sale_commission_delegated_partner 14.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

Commit:
[UPD] Update sale_commission_agent_restrict.pot

OCA-git-bot committed:
sale_commission_agent_restrict 14.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-14.0/commission-14.0-sale_commission_agent_restrict
Translate-URL: https://translation.odoo-community.org/projects/commission-14-0/commission-14-0-sale_commission_agent_restrict/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: commission-14.0/commission-14.0-sale_commission_agent_restrict
Translate-URL: https://translation.odoo-community.org/projects/commission-14-0/commission-14-0-sale_commission_agent_restrict/it/

hieulucky111 committed:
Merge branch '14.0' into sale_commission_formula-ready
